src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/SubscriptionHook.cs

```diff
@@ -0,0 +1,28 @@
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model for the subscription hook
```

Comment:
A subscription hook uses an existing T2 service to evaluate if the instanceOwnerId can instantiate the app.